timemachine/fe/single_topology.py

```diff
@@ -286,17 +312,49 @@ def setup_end_state(ff, mol_a, mol_b, core, a_to_c, b_to_c):
     all_dummy_bond_idxs, all_dummy_bond_params = [], []
     all_dummy_angle_idxs, all_dummy_angle_params = [], []
     all_dummy_improper_idxs, all_dummy_improper_params = [], []
+    all_dummy_chiral_atom_idxs, all_dummy_chiral_atom_params = [], []
 
     dgs = find_dummy_groups_and_anchors(mol_a, mol_b, core[:, 0], core[:, 1])
     # gotta add 'em all!
+
+    # use mol_b to find chiral_atom_idxs
+    mol_b_top = topology.BaseTopology(mol_b, ff)
+    mol_b_chiral_atom, _ = mol_b_top.setup_chiral_restraints()
+    mol_b_chiral_atom_idxs = mol_b_chiral_atom.potential.idxs
+
+    # (i,j,k) angle idxs implied by (center, i, j, k)
+    interpolating_mol_b_chiral_atom_angle_idxs = set()
+    for center, i, j, k in mol_b_chiral_atom_idxs:
+        # if this chiral atom restraint is present in mol_b but not in the dummy group,
+        # then it is being interpolated and the angle terms need to be weakened
+        if (center, i, j, k) not in all_dummy_chiral_atom_idxs:
+            interpolating_mol_b_chiral_atom_angle_idxs.add(canonicalize_bond([i, center, j]))
+            interpolating_mol_b_chiral_atom_angle_idxs.add(canonicalize_bond([i, center, k]))
+            interpolating_mol_b_chiral_atom_angle_idxs.add(canonicalize_bond([j, center, k]))
+
     for anchor, (nbr, dg) in dgs.items():
         all_idxs, all_params = setup_dummy_interactions_from_ff(ff, mol_b, dg, anchor, nbr)
         all_dummy_bond_idxs.extend(all_idxs[0])
         all_dummy_angle_idxs.extend(all_idxs[1])
         all_dummy_improper_idxs.extend(all_idxs[2])
+        all_dummy_chiral_atom_idxs.extend(all_idxs[3])
+
         all_dummy_bond_params.extend(all_params[0])
-        all_dummy_angle_params.extend(all_params[1])
+
+        # weaken angle terms defining the chiral restraints
+        modified_angle_params = []
+        for angle_idxs, angle_params in zip(all_idxs[1], all_params[1]):
+            # sanity check that we're already canonical
+            assert canonicalize_bond(angle_idxs) == angle_idxs
+            if angle_idxs in interpolating_mol_b_chiral_atom_angle_idxs:
+                modified_angle_params.append([MINIMUM_CHIRAL_ANGLE_FORCE_CONSTANT, angle_params[1]])
+            else:
+                modified_angle_params.append(angle_params)
+
+        modified_angle_params = np.array(modified_angle_params)
+        all_dummy_angle_params.extend(modified_angle_params)
         all_dummy_improper_params.extend(all_params[2])
+        all_dummy_chiral_atom_idxs.extend(all_params[3])
```

Comment:
another typo I introduced in https://github.com/proteneer/timemachine/pull/1255/commits/e44f4e7788975d17e85494d3dd59c976b42c26b9
```suggestion
        all_dummy_chiral_atom_params.extend(all_params[3])
```